providers/mailtrap/jest.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  moduleNameMapper: {
+    axios: 'axios/dist/node/axios.cjs',
```

Comment:
without that option Jest has problem with importing `axios` inside [Mailtrap SDK](https://github.com/railsware/mailtrap-nodejs).